pkg/osquery/table/mdm.go

```diff
@@ -36,13 +71,18 @@ func generateMDMInfo(ctx context.Context, queryContext table.QueryContext) ([]ma
 		return nil, err
 	}
 
-	depEnrolled, userApproved := "unknown", "unknown"
+	depEnrolled, depCapable, userApproved := "unknown", "unknown", "unknown"
 	status, err := getMDMProfileStatus()
 	if err == nil { // only supported on 10.13.4+
 		depEnrolled = strconv.FormatBool(status.DEPEnrolled)
 		userApproved = strconv.FormatBool(status.UserApproved)
 	}
 
+	depstatus, err := getDEPStatus()
+	if err == nil {
```

Comment:
if err != nil, return it. The logic here doesn't make much sense because the error is ignored.